### ISSUE TITLE
Question: Does Mob-Stacker remove visible monsters/villagers other entities altogheter?

### ISSUE THREAD

**Issue by exocyt0sis** [frikinjay/mob-stacker#12]
While this mod seems like really useful for clients and servers alike, I don't understand how it's implemented. When having Mob Stacker installed, does this mean the Minecraft world will have fewer _visible_ entities (monsters, villagers and so on) in the world? For instance, if a horde of 10 zombies would attack without the mod, will Mob Stacker reduce the number of attacking zombies when installed?

**Comment by frikinjay** [comment]
Yes, the mob will put those 10 zombies into 1 zombie but that one zombie can be killed 10 times and the loot will drop for each time its killed, there's a config option where that one zombies health will be scaled to that of 10 zombies and when it dies it will drop the loot of 10 zombies all together. So instead of the machine processing 10 zombies it only had to process one at a time while still getting the loot of 10 zombies
